python/cudf/cudf/core/multiindex.py

```diff
@@ -158,7 +159,7 @@ def __init__(
                     "codes and is inconsistent!"
```

Comment:
FWIW, I am not sure that `codes` can be `np.int64` it should probably be size_type_dtype. But let us not fix that here.